### PR TITLE
telemetry sdk name changed to lm-opentelemetry

### PR DESCRIPTION
**Pull request by saytoshivam**

### CODE REVIEW

File: otel-resource/src/main/java/com/logicmonitor/resource/detectors/common/LMCommonResource.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class LMCommonResource {
+  private static final Resource INSTANCE = buildResource();
+
+  public static Resource get() {
+    return INSTANCE;
+  }
+
+  static Resource buildResource() {
+    AttributesBuilder attrBuilders = Attributes.builder();
+    String telemetrySdkName = "lm-opentelemetry";
```
Comment (Gaurang-Patel) [inline otel-resource/src/main/java/com/logicmonitor/resource/detectors/common/LMCommonResource.java:17]: Maybe we can pass directly to attrBuilders.put

**Comment by Gaurang-Patel** [comment]
looks good to me .

**Review by Gaurang-Patel — APPROVED** [review]

File: otel-resource/src/main/java/com/logicmonitor/resource/detectors/common/LMCommonResource.java (hunk 1 above)
Comment (saytoshivam) [reply otel-resource/src/main/java/com/logicmonitor/resource/detectors/common/LMCommonResource.java:17]: we can pass, but it was informed to me that first i need to set it in variable , then use that variable in argument to put method

Comment (saytoshivam) [reply otel-resource/src/main/java/com/logicmonitor/resource/detectors/common/LMCommonResource.java]: optimised

**Review by DondekarShraddha — APPROVED** [review]

**Pull request merged**